declarations_site/spotter/templatetags/spotter_tags.py

```diff
@@ -9,6 +9,8 @@ def uk_plural(value, arg):
     args = arg.split(',')
     value = int(value)
     rem = value % 10
+    if value > 100:
```

Comment:
cause it's two different apps, otherwise we should make a third common library, but for only one function ufff.